### PR TITLE
DDBTEAM-389: Helm chart importers

### PR DESCRIPTION
**Pull request by cableman**

### CODE REVIEW

File: .github/workflows/tag.yml [hunk 1]
```diff
@@ -35,22 +35,21 @@ jobs:
...
     steps:
       - uses: actions/checkout@master
-      
-      # Set the target Azure Kubernetes Service (AKS) cluster. 
-      - name: Set AKS Context (Prod)
-        uses: azure/aks-set-context@v1
+
+      # Set the target Azure Kubernetes Service (AKS) cluster.
+      - uses: azure/aks-set-context@v1
+        name: Set AKS Context (stg)
```
Comment (tuj) [inline .github/workflows/tag.yml]: Shouldn't this be Prod?

File: .github/workflows/tag.yml [hunk 2]
```diff
@@ -35,22 +35,21 @@ jobs:
...
+        if: github.ref == 'refs/heads/release'
         with:
-          creds: '${{ secrets.AZURE_CREDENTIALS }}'
-          cluster-name: ddb-cover-service
-          resource-group: CoverService
+          creds: '${{ secrets.AZURE_CREDENTIALS_PROD }}'
+          cluster-name: ddb-cover-service-stg
```
Comment (tuj) [inline .github/workflows/tag.yml]: -stg ?

File: .github/workflows/tag.yml (hunk 1 above)
Comment (cableman) [reply .github/workflows/tag.yml]: Fixed

File: .github/workflows/tag.yml (hunk 2 above)
Comment (cableman) [reply .github/workflows/tag.yml]: Fixed

**Pull request merged**